### DISCUSSION TITLE
Web Page Information Correction

### DISCUSSION THREAD

**Discussion by ghost** [mdn/mdn-community#220] [Content] [answered]
https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Media_queries

Hi, the sample with `@media screen and (max-width: 600px) {...` is described as "...if viewport is narrower than 600 pixels...". This should be "...if viewport is 600 pixels or narrower..." (or similar language that includes 600px itself in the range).

**Comment by teoli2003** [comment, marked as answer]
Thank you! I've created https://github.com/mdn/content/pull/20152 to fix the issue.

This is more of an issue than something to discuss. Next time, can you open an issue instead? It would be easier. Or even better, just open a PR for such a trivial change.

Again, thank you for the report.

**Reply by teoli2003** [reply]
The fix was merged overnight. It should be live later today. I think we can close this discussion.

**Comment by ghost** [comment]
Dear Jean-Yves, I noticed the error on the normal mdn documentation site. From there, I saw no information or links to open an issue or PR. If it is there, it is not readily apparent. When I came here, my intention was indeed to open an issue but again the link for that or where to do that was not readily apparent. 

I suggest open another issue indicating a.) need to update the main mdn doc site to have a clear path from there to opening an issue and b.) update the mdn github site to also have readily apparent info on where to open an issue. 

I only found the discussion area and I did look for where to open an issue once I got here from the main mdn doc site.

Thank You

**Reply by teoli2003** [reply]
Thanks again for your help!  We have a box with the relevant links at the bottom of each page.

![Capture d’écran 2022-08-31 à 17 55 56](https://user-images.githubusercontent.com/1466293/187723859-cf4826c3-b16f-49b6-8db5-dccd74514e87.png)

**Comment by ghost** [comment]
Good to know. I suggest place that on the right-hand side below "In this article". The bottom is far down sometimes so perhaps that's the "not readily apparent" aspect.  :-)

**Comment by ghost** [comment]
I saw the update. Sounds great. Thanks for additional help information. I also am happy to be helpful and hope that I was.
